velox/vector/fuzzer/VectorFuzzer.h

```diff
@@ -44,14 +44,17 @@ enum UTF8CharList {
 ///
 /// #1.
 ///
-/// The `fuzz(type)` method provides the highest degree of entropy. It randomly
-/// generates different types of (possibly nested) encodings given the input
-/// type, including constants, dictionaries, sliced vectors, and more. It
-/// accepts any primitive, complex, or nested types:
+/// The `fuzz(type, canBeLazy)` method provides the highest degree of entropy.
+/// It randomly generates different types of (possibly nested) encodings given
+/// the input type, including constants, dictionaries, sliced vectors, and more.
+/// It accepts any primitive, complex, or nested types. Additionally,
+/// 'canBeLazy' can be set to true to allow a 50% chance of generating a lazy
```

Comment:
Probably need to update this comment now. Maybe just remove the actual percentage from this comment as it will very likely be outdated once the next person tweaks the internal behavior.